networking_aci/tests/unit/plugins/ml2/drivers/mech_aci/test_driver.py

```diff
@@ -23,27 +24,28 @@ def setUp(self):
         super().setUp()
         self._register_azs()
         ctx = context.get_admin_context()
-        with ctx.session.begin(subtransactions=True):
+        with db_api.CONTEXT_WRITER.using(ctx):
```

Comment:
So I see you're replacing the `ctx-session.begin` with the proper Neutron-way of using `db_api` - is this only a problem in the tests or do we need now with caracal to do that outside the tests as well?

```
quoth:~/repos/networking-aci/networking_aci$ git curr
stable/2024.1-m3
quoth:~/repos/networking-aci/networking_aci$ git grep session.begin
plugins/ml2/drivers/mech_aci/allocations_manager.py:            with session.begin(subtransactions=True):
plugins/ml2/drivers/mech_aci/allocations_manager.py:        with db_api.exc_to_retry(sa.exc.IntegrityError), session.begin(subtransactions=True):
plugins/ml2/drivers/mech_aci/allocations_manager.py:        with session.begin(subtransactions=True):
plugins/ml2/drivers/mech_aci/allocations_manager.py:        with session.begin(subtransactions=True):
plugins/ml2/drivers/mech_aci/allocations_manager.py:        with db_api.exc_to_retry(sa.exc.IntegrityError), session.begin(subtransactions=True):
plugins/ml2/drivers/mech_aci/allocations_manager.py:        with session.begin(subtransactions=True):
plugins/ml2/drivers/mech_aci/common.py:        with context.session.begin(subtransactions=True):
plugins/ml2/drivers/mech_aci/common.py:        with context.session.begin(subtransactions=True):
tests/unit/plugins/ml2/drivers/mech_aci/test_driver.py:        with ctx.session.begin(subtransactions=True):
tests/unit/plugins/ml2/drivers/mech_aci/test_driver.py:                with ctx.session.begin():
```